README.md

```diff
@@ -57,3 +57,5 @@ docker-compose -f test/docker-compose.v8.overrides.yml -f test/docker-compose.v8
 
 - https://github.com/elastic/go-elasticsearch
 - https://www.elastic.co/guide/en/elasticsearch/reference/7.17/docs-bulk.html
+
+![scarf pixel](https://static.scarf.sh/a.png?x-pxid=1423de19-24e7-4d64-91cf-0b893ca28as7)
```

Comment:
@maha-hajja IIRC, the Scarf pixels go at the top, below the header right?

Reply:
No, if we put it below the header we get a new empty line that looks awkward. This is fine.

That said, the scarf pixel should be different for every connector, I created a separate one for this connector @parikshitg `https://static.scarf.sh/a.png?x-pxid=715ebf4a-148c-44ad-8f64-6cc5780d34ae`.

Reply:
updated.